src/engine/server/server.cpp

```diff
@@ -829,14 +829,6 @@ void CServer::ProcessClientPacket(CNetChunk *pPacket)
 					return;
 				}
 
-				const char *pPassword = Unpacker.GetString(CUnpacker::SANITIZE_CC);
```

Comment:
why is the password check deleted?

Reply:
I guess it moved somewhere else. Don't get bamboozled by me being the author of this pr. This is a 11 year old stale pr by Choupom. I do not know why oy never closed it. But we should probably close it. Since it is version breaking.